docs/user-guide/system-admin/configuration/installation.md

```diff
@@ -120,8 +114,8 @@ Any other value will be treated as the name of a virus scanner command: the comm
 
 Go to the `<JBOSS>/bin` directory and run
 
-* `standalone.sh` or `start-zanata.sh` for Linux, Mac
-* `standalone.bat` or `start-zanata.bat` for Windows
+* `standalone.sh -c standalone-zanata.xml` or `start-zanata.sh` for Linux, Mac
```

Comment:
This is so that they can use the new `zanata.properties` file to set configuration.